Makefile

```diff
@@ -112,16 +120,30 @@ manifest-templator: $(GO)
 
 docker-build: docker-build-operator docker-build-registry
 
-docker-build-operator: manager manifest-templator
-	$(OCI_BIN) build -f build/operator/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
+docker-build-operator:
+ifeq ($(OCI_BIN),podman)
+	$(MAKE) build-multiarch-operator-image
+else ifeq ($(OCI_BIN),docker)
+	$(MAKE) create-builder
+	docker buildx build \
+        --build-arg BUILD_ARCH=$(ARCH) \
+        --platform $(PLATFORMS) \
+        -f build/operator/Dockerfile \
+        -t ${OPERATOR_IMAGE_TAGGED} \
+        --push .
```

Comment:
please consider moving it under create-builder target
unless you think it doesnt fit there?